src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -159,6 +159,8 @@
     private final YesOrNo smallClaimHearingInterpreterRequired;
     private final String smallClaimHearingInterpreterDescription;
 
+    private final String additionalInformationForJudge;
```

Comment:
Is there a reason why this is not part of new field FutureApplications in Applicant1DQ?